pkg/controller/installation/products/threescale/asserts.go

```diff
@@ -0,0 +1,86 @@
+package threescale
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	threescalev1 "github.com/integr8ly/integreatly-operator/pkg/apis/3scale/v1alpha1"
+	aerogearv1 "github.com/integr8ly/integreatly-operator/pkg/apis/aerogear/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/controller/installation/products/rhsso"
+	oauthClient "github.com/openshift/client-go/oauth/clientset/versioned/typed/oauth/v1"
+	coreosv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	k8serr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	pkgclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func assertInstallationSuccessfullyReconciled(installation *v1alpha1.Installation, fakeSigsClient *SigsClientInterfaceMock, fakeThreeScaleClient *ThreeScaleInterfaceMock, fakeOauthClient oauthClient.OauthV1Interface) error {
+	ctx := context.TODO()
```

Comment:
we shouldn't be summoning up contexts from nowhere anymore, let's pass in the root context

Reply:
oh wait, this is to do with unit tests? In that case it's fine!

Reply:
let's rename the file `_test.go`?